server/models/electionModel.js

```diff
@@ -0,0 +1,56 @@
+const mongoose = require('mongoose');
+
+//To create election
+// 1. questionsArray : question, Indexes Id, options
+// 2. electionTitle
+// 3. Status of selection (For this election Id, do we meed the minimum registered users? If so
+// 4. Voting Started(true / false) -> Will be sent from FE once voter starts elections after eligibility is met
+// export const REGISTRATION_STATUS = "Registeration Open";
+// export const VOTING_IN_PROGRESS_STATUS = "Voting Started";
+// export const VOTING_ENDED_STATUS = "Voting Ended";
+// array of collector ids
+// electionCreatorProfileId
+
+const electionSchema = mongoose.Schema (
+    {   
+        questions: [ {
+            question : {
+                type: String,
+                required: true
+            },
+            options: [ {
+                option : {
+                    type: String,
+                    required: true
+                },
+            questionId: mongoose.Schema.Types.ObjectId,
```

Comment:
this is in the wrong place. This needs to be moved up.

Reply:
Have changed it now with the latest commit